src/ucentral-client/platform/brcm-sonic/gnma/gnmi/gnmi_c_connector.cpp

```diff
@@ -354,12 +362,20 @@ static int gnmi_jsoni_get_internal(struct gnmi_session *gs, const char *path,
 	gpath = greq.add_path();
 	convertYangPath2ProtoPath(path, gpath);
 
+	::grpc::ClientContext context;
+	set_deadline_after_us(context, timeout_us);
+	status = (*gs->stub)->Get(&context, greq, &gres);
+
+	/*
+	 * We don't have the implementation of JWT authentication, so disable it
```

Comment:
You shouldn't be changing implementation of existing platform;
I'm still looking through code, but if you're using this implementation in Larch-plat code as well, we have to come up with some layering to maybe separate potential common code or something like that